### DISCUSSION TITLE
NCIRF not running on Windows 11

### DISCUSSION THREAD

**Discussion by georgstamm** [ncidose/ncidosetools#21] [NCIRF] [answered]
I tried to install NCIRF2.0.20230812_windows.exe on my Windows 11 laptop. But I get the error message:
"Your operating system is not supporte". Any hints?

**Comment by choonsiklee** [comment]
It seems that there's some problem with packaging process. I will get back to you shortly once the issue is resolved.

**Reply by choonsiklee** [reply, marked as answer]
The issue is now resolved and the new installation file has been tested on multiple Windows 11 machines. Please try again!

**Comment by Alexi2686** [comment]
I'm encountering an error message when attempting to click "Calculate dose" in NCIRF 2.0.20230812. The error message reads: "An exception of class IOException was not handled. The application must shut down. Exception Error Number: 2."

NCIRF is installed on a Windows 11 laptop. Interestingly, there are no such errors on Windows 10. I've tried compatibility mode and running as an administrator, but unfortunately, the issue persists.

**Reply by choonsiklee** [reply]
Hello, we have tested the version 2.0.202330812 on three different Windows 11 machines and confirmed that it works fine. The error you reported has nothing to do with Windows version. Can you try to install the program on other Windows 11 machines?

**Reply by Alexi2686** [reply]
Hello, unfortunately, I only have access to one PC with Windows 11 to install NCIRF. I will change some Windows settings to see if it solve the issue.
